pages/vi/vi-docker-tutorial.md

```diff
@@ -68,11 +68,9 @@ In the [previous step]( http://open-learning-exchange.github.io/#!./pages/vi/vi-
   
 5. Run the *following command* to spawn your environment for the **first time**:
 
-    ```
-    ATTENTION: if you followed Step1 and configured Planet, you should not run `docker-compose -f planet.yml -p planet up -d --build`.
-    It might destroy your configuration.
-    `vagrant up prod` runs `docker-compose -f planet.yml -p planet up -d --build` automatically.
-    ```
+    
+WARNING: If you followed Step1 and configured Planet, you should not run `docker-compose -f planet.yml -p planet up -d --build`. It might destroy your configuration. `vagrant up prod` runs `docker-compose -f planet.yml -p planet up -d --build` automatically. If you are in this situation, look at the ** [Second and third element of Troubleshooting in this page]( https://open-learning-exchange.github.io/#!./pages/vi/vi-configurations-vagrant.md#Troubleshooting)** 
```

Comment:
do we need this `--build` cause I think we don't

Reply:
@dogi Thanks for comments. I did not change the docker code here, just change the background format. All the instructions in this [docker tutorial](https://open-learning-exchange.github.io/#!pages/vi/vi-docker-tutorial.md) include the `--build` to start the container. I googled the difference, and [here](https://stackoverflow.com/questions/39988844/docker-compose-up-vs-docker-compose-up-build-vs-docker-compose-build-no-cach) showed `--build` would build unneeded images. If you want me to remove the `--build`, I can remove it in this warning or all of them in this [page](https://open-learning-exchange.github.io/#!pages/vi/vi-docker-tutorial.md).

Reply:
@lmmrssa what is your opinion?

Reply:
@dogi yes `--build` is not required @wanglihe2010 we are not saying you added code. But as you were working on this section and we noticed that there is code that is not required so we wanted that to be removed. Because now with WARNING people will have close eye on this command.

Reply:
@dogi @lmmrssa I just deleted `--build` in the WARNING. To be consistent, I also deleted two more `--build` of same command in this page, one is right below the WARNING, another is in the [Docker Compose](https://raw.githack.com/wanglihe2010/wanglihe2010.github.io/issue-2633/#!pages/vi/vi-docker-tutorial.md#Docker_Compose) section. Please let me know whether I did it right. Thanks